app/assets/stylesheets/views/article.scss

```diff
@@ -460,22 +460,26 @@
 }
 
 .multiple_reactions_aggregate {
-  padding-right: 1em;
+  display: flex;
+  align-items: center;
 
   .crayons_icon_container {
     display: inline-block;
     background-color: var(--body-bg);
+    border: solid;
```

Comment:
(suggestion): You could use the [border css shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/border) here:

```
border: 2px solid var(--card-bg)
```

and remove `border: solid;`, ` border-color: var(--card-bg);`, `border-width: 2px;`

Reply:
Done. Thanks.